blazingcache-core/src/main/java/blazingcache/client/EntryHandle.java

```diff
@@ -28,17 +28,18 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.lang.ref.SoftReference;
-import java.util.Arrays;
-import java.util.logging.Level;
 import java.util.logging.Logger;
 
 /**
- * Una entry nella cache
+ * An handle to an entry.
+ * <p>
+ * Entries are reference counted so you have to call explicitly {@link #close() }
+ * in order to release the resources associated to the entry.
  *
  * @author enrico.olivelli
  */
 @SuppressFBWarnings({"EI_EXPOSE_REP2", "EI_EXPOSE_REP"})
-public final class CacheEntry implements AutoCloseable {
+public final class EntryHandle implements AutoCloseable {
```

Comment:
Just to be cleaner toString method too should be changed

Reply:
good catch.
resovled